Gemfile

```diff
@@ -22,6 +22,7 @@ gem "pry", "~> 0.13.1"
 gem "puma", "~> 4.1"
 gem "rails", "~> 6.0.3", ">= 6.0.3.2"
 gem "rails-i18n"
+gem "ransack"
```

Comment:
nêu ngắn gọn chức năng chính của gem này

Reply:
Ransack là 1 gem trong Ruby on the Rails giúp hỗ trợ người dùng xây dựng các tìm kiếm.
Ransack cho phép tìm kiếm với hai hình thức đơn giản và nâng cao (phần nâng cao này mình vẫn chưa rõ lắm )
Cái ransack này nó còn giúp sắp xếp cái kết quả tìm kiếm